src/cli/action_convert.cpp

```diff
@@ -287,6 +288,11 @@ bool ActionConvert::process_file(
 	m_width = opts.get<int>(opts::width);
 	m_height = opts.get<int>(opts::height);
 
+	if (!std::filesystem::exists(srcFile)) {
+		std::cerr << "Could not open " << srcFile << ", file does not exist\n";
```

Comment:
Can you use a colon `:` instead of a comma here? It would be more consistent with other error messages.